routes/web.php

```diff
@@ -41,6 +41,10 @@
         Route::get('orders', 'OrderController@index')->name('orders.index');
         Route::post('orders/remove', 'OrderController@removeProductInCart')->name('orders.product.remove');
         Route::post('orders', 'OrderController@addToCart')->name('orders.addToCart');
+        Route::post('orders/plusQuantity', 'OrderController@increaseQuantity')->name('orders.increaseQuantity');
```

Comment:
```suggestion
        Route::post('orders/plus-quantity', 'OrderController@increaseQuantity')->name('orders.increaseQuantity');
```
url ngta hay đặt theo dạng này chứ ko đặt theo dạng camelCase đâu nhé 
với lại để đồng bộ increase decrease đi e ơi